src/common/darktable.c

```diff
@@ -1911,6 +1913,8 @@ int dt_init(int argc, char *argv[], const gboolean init_gui, const gboolean load
     if(changed_xmp_files)
       dt_control_crawler_show_image_list(changed_xmp_files);
 
+    gtk_widget_show_all(dt_ui_main_window(darktable.gui->ui));
+    // give Gtk a chance to actually process the resizing
```

Comment:
referred to process_events which doesn't seem to be needed here.

Reply:
right, misleading comment